superflore/generators/bitbake/ros_meta.py

```diff
@@ -31,24 +31,51 @@ def __init__(
 
     def clean_ros_recipe_dirs(self, distro=None):
         if distro:
-            info('Cleaning up recipes-ros-{0} directory...'.format(distro))
-            self.repo.git.rm('-rf', 'recipes-ros-{0}'.format(distro))
+            info(
+                'Cleaning up generated-recipes-{} directory...'.format(distro))
+            self.repo.git.rm('-rf', 'generated-recipes-{}'.format(distro))
         else:
-            info('Cleaning up recipes-ros-* directories...')
-            self.repo.git.rm('-rf', 'recipes-ros-*')
+            info('Cleaning up generated-recipes-* directories...')
+            self.repo.git.rm('-rf', 'generated-recipes-*')
 
     def commit_changes(self, distro):
         info('Adding changes...')
         if distro == 'all':
             commit_msg = 'regenerate all distros, {0}'
-            self.repo.git.add('recipes-ros-*')
+            self.repo.git.add('generated-recipes-*')
+            self.repo.git.add(
+                'conf/ros-distro/include/*/generated-ros-distro.inc')
+            self.repo.git.add('files/*/cache.*')
+            self.repo.git.add('files/*/rosdep-resolve.yaml')
+            self.repo.git.add('files/*/newer-platform-components.list')
+            self.repo.git.add('files/*/superflore-change-summary.txt')
         else:
             commit_msg = 'regenerate ros-{1}, {0}'
-            self.repo.git.add('recipes-ros-{0}'.format(distro))
+            self.repo.git.add('generated-recipes-{0}'.format(distro))
+            self.repo.git.add(
+                'conf/ros-distro/include/{0}/generated-ros-distro.inc'
+                .format(distro))
+            self.repo.git.add('files/{0}/cache.*'.format(distro))
+            self.repo.git.add('files/{0}/rosdep-resolve.yaml'.format(distro))
+            self.repo.git.add(
+                'files/{0}/newer-platform-components.list'.format(distro))
+            self.repo.git.add(
+                'files/{0}/superflore-change-summary.txt'.format(distro))
```

Comment:
@andre-rosa What are these things you're adding to the repo here? I'm a bit confused.

Reply:
They're files containing meta-data about the changes in the state of upstream ROS components. Those will really help on OpenEmbedded maintenance by clearly stating what were the changes between any major/minor releases, so that one can expect changes to any existing overrides.